projects/canopy/src/lib/forms/radio/radio-button.component.html

```diff
@@ -6,10 +6,15 @@
   [attr.name]="name"
   [attr.value]="value"
   [attr.aria-invalid]="errorClass"
+  [attr.aria-describedby]="ariaDescribedBy || null"
   class="lg-radio-button__input"
   type="radio"
 />
 
 <label [attr.for]="id" class="lg-radio-button__label">
   <ng-content></ng-content>
 </label>
+
+<div *ngIf="ariaDescribedBy" class="lg-radio-button__hint">
```

Comment:
Just wondering why we need `*ngIf="ariaDescribedBy"`? From a quick look at the code, it seems that we set `aria-describedby` based on the id of the hint component. So maybe this condition doesn't need to be here? E.g. we just display the hint and then set the aria attribute accordingly?

Reply:
Oh I see in other comments that you added conditional logic so that you don't display the div wrapper if the hint isn't used. It does still seem odd that you'd tie this to the `ariaDescribedBy` attribute though, could we use the existence of the hint component instead?

Reply:
Indeed 😄 . I've noticed that this is a way that doesn't require checking for template referencing where we would potentially have to view the child and then do extra checks on it. I figured we could use a value from an existing input (`ariaDescribedBy`) that we have. Still, I can understand if that seems odd and if a different approach is preferred, I'm down to change it like so 😉 .

Reply:
My instinct is to try getting a reference to the hit component - maybe try that and see if it isn't too cumbersome? I'm thinking that we should avoid using `ariaDescribedBy`, as the developer might choose to set this independently and not use the hint component inside the radio button component.

Reply:
Yes, I agree with @owensgit 👍